include/database.py

```diff
@@ -59,3 +59,69 @@ def insert_returning(connectionString, sql, parameters):
     except psycopg2.DatabaseError as err:
         sys.stderr.write(f'A general database error occurred: {err}')
         raise DatabaseError
+
+
+def remove_record(connectionString, sql, parameters):
```

Comment:
`delete` might be sufficient in and of itself since that follows the SQL nomenclature